modules/doi/modules/doi_importer/includes/importer.inc

```diff
@@ -24,38 +24,44 @@ class DOIImporter extends IslandoraBatchImporter {
    *
    * @see IslandoraBatchImporter::getForm()
    */
-  public static function getForm(array &$form_state) {
-    return array(
-      'fs' => array(
-        '#type' => 'fieldset',
-        '#title' => t('DOI Batch Journal Importer'),
-        '#description' => t('Provide either a list of Digital Object Identifiers in a file, or enter DOIs in the textarea below. In either case, the DOIs in the list should be separated by either whitespace or commas (or some combination thereof). NOTE: Presently, only journal (articles) are supported.'),
-        'file' => array(
-          '#type' => 'managed_file',
-          '#title' => t('File of DOIs'),
-          '#description' => t('A file listing Digital Object Identifiers (with ".txt" extension).'),
-          '#upload_validators' => array(
-            'file_validate_extensions' => array('txt'),
+  public static function getForm(array & $form_state) {
+    $openurl_pid = variable_get('islandora_doi_openurl_pid', 'user@example.com');
```

Comment:
@DiegoPino It is NOT something Scholar users are aware of, since both of the current module maintainers (@DonRichards  and I) encountered the same issue where we expected the DOI Importer to work out of the box and didn't know you had to register an email address and add it in the DOI config admin menu.

This is the primary reason why I chose to have the form not load if the email is set to the default; if the email is the default, they probably don't know that they even need to change it. The error message that gets displayed has a link to the page where they can set that configuration, and that page has more info how to set up a CrossRef registered email account.

Theres the chance that they could be using an email address other than the default which is not a registered CrossRef email, but we really have no way of checking that and if they enter a custom invalid email address than its their own fault ;)